### DISCUSSION TITLE
T4 generated code are not sorted alphabetically

### DISCUSSION THREAD

**Discussion by ndc** [linq2db/linq2db#4367] [Q&A]
Hello,

I have a problem using the T4 template of `linq2db.SqlServer` version 5.3.2. The generated code from my machine is different from my colleague's. Looks like it is sorted alphabetically in my colleague's machine but not sorted in my machine, but only for certain areas.
For example the tables are sorted alphabetically both in my machine and my colleague's.
But in 'Table Functions' I have a function called 'DelimitedSplit8K' that appears before 'CommaSeparatedListToTable' while in my colleague's machine it's the other way around.
Stored procedures and SqlFunctions are not ordered alphabetically either in my machine while they are in my colleague's machine.
TableExtensions are ordered alphabetically both in my machine and my colleague's.

This behavior makes it look like I change a lot of things in my commit.

Is it possible to add a command in the template so all parts are ordered alphabetically?

**Comment by igor-tkachev** [comment]
#4375
